src/app/components/app-build-details/app-build-details.component.html

```diff
@@ -24,7 +24,7 @@ <h1 class="bold">
                       </span>
                     </h1>
                   </div>
-                  <div class="column is-2">
+                  <div class="column is-2" *ngIf="build?.repository?.permissions[0]?.permission">
```

Comment:
this is very rough written `build?.repository?.permissions[0]?.permission`, I mean the `permissions[0]?.permission` part... can you reduce that in code to just `hasPermission` `boolean` value?